src/ekobox/info_file.py

```diff
@@ -39,7 +39,7 @@ def build(
     template_info["Authors"] = ""
     template_info["FlavorScheme"] = "variable"
     template_info.update(info_update)
-    template_info["NumFlavors"] = 14
+    template_info["NumFlavors"] = theory_card.heavy.num_flavs_max_pdf
```

Comment:
Same as https://github.com/NNPDF/nnpdf/pull/1746#discussion_r1219959665
```suggestion
    template_info["NumFlavors"] = max(nf for _, nf in op.mugrid)
```

Reply:
> PS: @AleCandido I know you want to give up on that specific variable (did we do it already?), so we would need to provide an alternative

This is the alternative. If really needed (also outside LHAPDF), we can make a property for this.

Reply:
I have the feeling LHAPDF metadata are rather redundant, but maybe I'm wrong...

Reply:
Why using the loop if we already have the property?

Reply:
> Why using the loop if we already have the property?

We don't have the property, the two things are not synced: one is a theoretical maximum declared in the `theory_card`, the other one is the one computed in practice, spelled out in the `op`erator card.
Moreover, the theoretical maximum is deprecated, since it is redundant with matching scales, so it is going to be removed in the next minor (i.e. breaking) release of EKO, v0.14.0.

Reply:
Before applying, I'll wait for @felixhekhorn to take a look and confirm

Reply:
We should keep this in mind for the new theory database (whenever we actually do it) since there we do have a maximum set.

Reply:
I hope that the new database (if ever) will be almost straightforward, after this round of polishing the pineline.

EKO has an almost minimal runcard, the more we refactor the internals, the more we clean it up. Same thing is going to happen to yadism, since I'd like to move the runcard translation to Pineko.
The moment we have minimal theories for each program, the theory database fields will be the union of the various runcards attributes (the only non-straightforward part is in Pinefarm, since we should propagate theory settings even to MadGraph and the other MCs).

Reply:
I guess this is also fine ... this makes the info file slightly more static (you can not add dynamically more Q2 points later on), but I guess that feature might be never needed